charmap.txt

```diff
@@ -304,4 +304,7 @@ YELLOW_4 = 44 @ D
 LIGHT_BLUE = 45 @ E
 WHITE_5 = 46 @ F
 LIGHT_BLUE_2 = 47 @ G (TODO CHECK THIS)
+ITEM_COLOR = 49 @ I
+CYAN_3 = 4D @ M
 YELLOW_5 = 4E @ N
+UNKNOWN_COLOR = 57 @ W
```

Comment:
From reading the text drawing functions, the colors are mapped this way:
```
	bytes 0x00 to 0x10 as is
	bytes 0x30 to 0x40: subtract 0x30
	C, D, N = 6
	E, S, T = 15
	G, M = 5
	H, I = 4
	W = 2
	other = 7
```
So, `UNKNOWN_COLOR` is red, not sure why you're calling that one `ITEM_COLOR` instead of green... Huh, actually that does make some kind of sense, but you might want to say it's green in the comment apart from just the letter it represents.
Would be nice to go through the contexts where the alternate colors are used and work out some better names.

Wouldn't it be better for all these macros to expand to `#C<color>` or `#c<color>` directly?

`#CW` seems to be used for errors specifically, strings like `#CW???#R`, `#CWinvalidity#R` and `#CWUnregistered dialog script#R`
`#CH` is used for friend area locations
`#CI` is items
`#CG` is used for mail and in mail-related context, also for link cable text.
`#CM` is always used for 'mon names
`#CT`, `#CE`, `#CS` are unused
`#CC` is used for debug menu items:
```
#CCScenario Progress Dummy#R
#CCScenario try dummy#R
#CCscene#R
#CCdirection#R
```
`#CD` for dungeon names. There are also a couple format strings with `#cD` but haven't looked into those.
`#CN` is fairly inconsistent, would need to look deeper to figure it out.
That should be all of them, I think.